test/unit/store/student-store.js

```diff
@@ -76,32 +81,84 @@ describe('StudentStore', () => {
             studentId: student.id,
             itemAddresses: [items[0].address, items[2].address]
         }).then(() => {
-            assert.include(student.items, items[0]);
+            assert.include(student.items[0].address, items[0].address);
             assert.notInclude(student.items, items[1]);
-            assert.include(student.items, items[2]);
+            assert.include(student.items[1].address, items[2].address);
```

Comment:
`include(string, string)` doesn't really make sense. It should be left as `include( [ object, object ], object)` or do string matching.